README.md

```diff
@@ -6,8 +6,82 @@
 
 [![GitHub Workflow Status (Testing)](https://img.shields.io/github/workflow/status/TrueBrain/TrueWiki/Testing/master?label=master)](https://github.com/TrueBrain/TrueWiki/actions?query=workflow%3ATesting)
 
-TrueWiki is a wikitext server similar to [mediawiki](https://github.com/wikimedia/mediawiki) and/or [gollum](https://github.com/gollum/gollum).
-As default storage it uses git (instead of a database).
+TrueWiki is an opinionated wikitext server similar to [mediawiki](https://github.com/wikimedia/mediawiki) and/or [gollum](https://github.com/gollum/gollum).
+As default storage it uses GitHub (instead of a database).
+
+## Opinionated
+
+TrueWiki iterates over what mediawiki offers to bring a more modern approach to a wiki.
+These opinions are described below.
+
+### Wikitext
+
+Although most of the wikitext is compatible with the one implemented by mediawiki, TrueWiki is a lot more strict and accepts less mistakes.
+
+For example, `[[ File:en/MyFile.png]]` is an error, as there should not be a space between `[[` and `File`.
+Other errors, like an unknown variable, links to pages that do not exist, etc, are also reported to the user.
+This in the hope to motivate people to both write better wikitext, but also to fix mistakes that otherwise go unnoticed.
+
+If there is one or more error, a badge is shown over "View Source", to inform anyone using the wiki that the page contains errors.
+
+### Translations
+
+Where mediawiki postfixes the language, TrueWiki prefixes them.
```

Comment:
Only the ottd wiki does that. Real wiki uses interwiki links.

Suggestion: While MediaWiki uses separate Wikis for each language, TrueWiki includes them in one.